public/app/plugins/datasource/postgres/specs/datasource.test.ts

```diff
@@ -1,85 +1,282 @@
-import { PostgresDatasource } from '../datasource';
+import { of } from 'rxjs';
+import { TestScheduler } from 'rxjs/testing';
+import { FetchResponse } from '@grafana/runtime';
 import { dateTime, toUtc } from '@grafana/data';
+
+import { PostgresDatasource } from '../datasource';
 import { backendSrv } from 'app/core/services/backend_srv'; // will use the version in __mocks__
 import { TemplateSrv } from 'app/features/templating/template_srv';
 import { initialCustomVariableModelState } from '../../../../features/variables/custom/reducer';
+import { TimeSrv } from '../../../../features/dashboard/services/TimeSrv';
+import { describe } from '../../../../../test/lib/common';
```

Comment:
Think you don't need this since describe is global?

Reply:
hmm, WebStorm is not behaving nicely. I'll fix